charts/os/cpu-stress/engine.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: cpu-stress--chaos
+spec:
+  engineState: 'active'
+  chaosServiceAccount: cpu-stress-sa
+  jobCleanUpPolicy: 'delete'
```

```suggestion
  jobCleanUpPolicy: 'retain'
```

We can remove this field - by default it is retain